### PR TITLE
Removed Gitter Chat widget with Discord Invite button

### LINKED ISSUE #75: Gitter Link still available on the front-end of the docs section of Plone.org that needs to be removed!
image

### PR DESCRIPTION
**Pull request by rohnsha0**
fixed #75

### CODE REVIEW

File: src/sphinxtheme/plone/plone_basic/layout.html [hunk 1]
```diff
@@ -295,16 +295,35 @@
...
+    position: fixed;
+    right: 30px;
+    bottom: 0px;
+
+    outline: turquoise;
+    border: none;
+    cursor: pointer;
+  }
+
+</style>
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/layout.html:320]: Let's make this a little more consistent with the existing button.

Also it should go into the style sheet, not inline in the template.

https://github.com/plone/sphinxtheme.plone/blob/a0ea9d4a1b7f87f5bb26efa828b70fd15f6d4f30/src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css#L100

```css
a.floating-discord-btn {
    background: #00a080cb;
    border-top-left-radius: 0.5em;
    border-top-right-radius: 0.5em;
    bottom: 0px;
    color: #fff;
    font-size: 12px;
    letter-spacing: 1px;
    padding: 1em 3em;
    position: fixed;
    right: 10px;
    text-decoration: none;
}
```

File: src/sphinxtheme/plone/plone_basic/layout.html [hunk 2]
```diff
@@ -295,16 +295,35 @@
...
+<form>
+  <a href="https://discord.gg/w8e5WCAKGs" target="_blank">
+    <input type="button" value="JOIN CHAT" class="floating-discord-btn">
+  </a>
+</form>
+
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/layout.html:326]: A form is not necessary. Just use an anchor.

```html
<a href="https://discord.gg/w8e5WCAKGs" target="_blank" class="floating-discord-btn">
JOIN CHAT
</a>
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: src/sphinxtheme/plone/plone_basic/layout.html (hunk 2 above)
Comment (rohnsha0) [reply src/sphinxtheme/plone/plone_basic/layout.html:326]: Just an anchor will do the thing?

File: src/sphinxtheme/plone/plone_basic/layout.html (hunk 1 above)
Comment (rohnsha0) [reply src/sphinxtheme/plone/plone_basic/layout.html:320]: Okay... I'll do this in comparison to our current button....

Comment (rohnsha0) [reply src/sphinxtheme/plone/plone_basic/layout.html:320]: Not applying the exact changes as declared in the stylesheet! @stevepiercy

Comment (rohnsha0) [reply src/sphinxtheme/plone/plone_basic/layout.html]: @stevepiercy please have a look at last commit... Is that working?

File: src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css [hunk 3]
```diff
@@ -98,3 +98,21 @@ div.admonition-description {
...
+  letter-spacing: 1px;
+  text-decoration: none;
+  color: white;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  font-size: 12px;
+  position: fixed;
+  right: 13px;
+  bottom: 0px;
+}
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css:118]: Please use the styles I provided in my earlier suggestion.

File: src/sphinxtheme/plone/plone_basic/layout.html [hunk 4]
```diff
@@ -102,6 +102,8 @@
   <link rel="icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" type="image/x-icon">
   <link rel="shortcut icon" type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
   <link type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
+  <link rel="stylesheet" type="text/css" href="static/css/ploneorg5.css">
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/layout.html:105]: Revert this. It is included through a configuration file at https://github.com/plone/sphinxtheme.plone/blob/a0ea9d4a1b7f87f5bb26efa828b70fd15f6d4f30/src/sphinxtheme/plone/plone_org_5/theme.conf.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by jensens — APPROVED** [review]

**Comment by rohnsha0** [comment]
@stevepiercy kindly re-verify the changes!

File: CHANGES.rst [hunk 5]
```diff
@@ -6,6 +6,7 @@ Changelog
 
 - Fix Travis setup to actually test Python 3.7/3.8 [loechel]
 - Algolia search css fix [polyester]
+- Removed Gitter Link from Front-End [rohnsha0]
```
Comment (stevepiercy) [inline CHANGES.rst:9]:
```suggestion
- Replaced link for chat button from Gitter to Discord [rohnsha0]
```

File: src/sphinxtheme/plone/plone_basic/layout.html [hunk 6]
```diff
@@ -102,6 +102,7 @@
   <link rel="icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" type="image/x-icon">
   <link rel="shortcut icon" type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
   <link type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
+
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/layout.html:105]: A blank line is also not needed. It is a good practice to restore code to its original when removing proposed changes to minimize the diff.

```suggestion
```

File: src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css [hunk 7]
```diff
@@ -98,3 +98,17 @@ div.admonition-description {
...
+  border-top-right-radius: 0.5em;
+  bottom: 0px;
+  color: #fff;
+  font-size: 12px;
+  letter-spacing: 1px;
+  padding: 1em 3em;
+  position: fixed;
+  right: 10px;
+  text-decoration: none;
+}
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css:114]: Add trailing carriage return to satisfy GitHub.

```suggestion
}

```

File: src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css [hunk 8]
```diff
@@ -98,3 +98,17 @@ div.admonition-description {
   border-color: #dedede;
   color: #086ca3;
 }
+
```
Comment (stevepiercy) [inline src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css:101]: I missed this earlier. The link needs a `hover` style.

```suggestion
a#floating-discord-btn:hover {
  background: #006b56;
}
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
Thank you! LGTM. Please go ahead and merge.

**Comment by rohnsha0** [comment]
> Thank you! LGTM. Please go ahead and merge.

@stevepiercy I can't as merging to restricted to authorized users only

**Pull request merged**